api/src/main/java/io/strimzi/api/kafka/model/JvmOptions.java

```diff
@@ -32,7 +32,7 @@
     private String xmx;
     private String xms;
     private Boolean server;
-    private boolean gcLoggingEnabled = false;
+    private boolean gcLoggingEnabled = true;
```

Comment:
Why was this changed?

Reply:
That was in @scholzj's original PR.

Reply:
@scholzj I was going to ask the same, why this?

Reply:
I am gonna revert this unless @scholzj has a strong objection?